client/components/happiness-support/index.jsx

```diff
@@ -44,17 +43,15 @@ export class HappinessSupport extends Component {
 		const components = {
 			strong: <strong />,
 		};
-		return preventWidows(
-			isJetpackFreePlan
-				? translate(
-						'{{strong}}Need help?{{/strong}} Search our support site to find out about your site, your account, and how to make the most of WordPress.',
-						{ components }
-				  )
-				: translate(
-						'{{strong}}Need help?{{/strong}} A Happiness Engineer can answer questions about your site and your account.',
-						{ components }
-				  )
-		);
```

Comment:
I'm not sure this component is still in use, but this text is in any case wrapped in a `p` tag so should be safe.

Reply:
Yeah I can't see it used anywhere, either.